portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -667,13 +667,74 @@ definition {
 	}
 
 	@description = "Verify that a Form can be created"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeCreated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-130500 CanBeCreated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		var text = "Test";
+
+		Form.editName(formName = "${text}");
+
+		Form.editDescription(formDescription = "${text}");
+
+		Form.editPageTitle(
+			formPageName = "${text}",
+			formTitleNumber = "1");
+
+		Form.editPageDescription(
+			formDescriptionName = "${text}",
+			formTitleNumber = "1");
+
```

Comment:
@ewertonluna Can you please change to "`Form 1`", "`Form 1 Description`", "`Page 1`", "`Page 1 Description`" instead of using "`Test`" for all the fields?